zipfs-law/bin/plotcounts_branch_keep.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import sys
+import re
+import argparse
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+
+def main(args):
+    """Run the command line program."""
+
+    input_csv = args.infile if args.infile else sys.stdin
```

Comment:
We need to explain somewhere that `pd.read_csv` takes either a stream or a string as its first argument - I only learned this recently.

Reply:
Agree - will definitely need to explain that in the text of the chapter.